src/components/News.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+<div class="row mt-md-5 mt-sm-5 mt-xs-5" >
+    <div class="col-lg-6 col-xs-12" v-for="(item,index) of this.info" :key="index">
+        <div clas="container-fluid" >
+        <div class="d-flex justify-content-center align-items-center">
+            <div class="p-3">
+                <img src="https://i.picasion.com/pic90/96f94ea318bfd9ede1d311621d6696f2.gif" width="200" height="200" border="0" alt="coffeeNews" >
+            </div>
+            <div class="p-2">
+               <p>{{ item.title }}</p>
+               <p>{{ item.description.slice(0,250) }}</p>
+               <p>{{ item.author }}</p>
+            </div>
+        </div>
+        <div class="d-flex container-fluid justify-content-end pb-2 mt-n2">
+            <button type="button" class="btn btn-outline-info" >
+               <a :href="item.url" >info</a>
+            </button>
```

Comment:
El link debería estar alrededor del botón, no al revés: como está, hay que dar click sobre la palabra para ir al href. El blank también es una buena práctica para abrir el link en una pestaña nueva y no sacar al usuario de la página.
```suggestion
            
               <a :href="item.url" target="_blank">
               <button type="button" class="btn btn-outline-info" >Info</button>
               </a>
            
```